MoreLinq/MoreEnumerable.cs

```diff
@@ -27,5 +27,18 @@ namespace MoreLinq
     
     public static partial class MoreEnumerable
     {
+        /// <summary>
```

Comment:
_Trivial_ private functions where the code speaks for itself means doc summaries feel redundant and add little value (even reads longer than the code) therefore best to remove it.

Reply:
I like having the Intellisense (I find the `out int` pattern to be more intuitive for this sort of thing, and others might as well), but I can do this.

Reply:
> I find the `out int` pattern to be more intuitive

I'm not sure I understand your reasoning here. There's no `out` variable here.

> I find…to be more intuitive…others might as well

It's best to avoid being subjective here and I can't speak for others. The things we should challenge are:

- Are method and argument names intuitive?
- Does the return type reflect the method name as best & succinctly as possible?
- The value of the documentation on top of everything that the code is communicating.

The IntelliSense argument is fair for public members but this is private, internal and trivial.

Reply:
> I'm not sure I understand your reasoning here. There's no out variable here.

This had to do with my assertion that `out int` was more idiomatic, which we concluded it was not already; I shouldn't have brought it up again. Sorry.

> The IntelliSense argument is fair for public members but this is private, internal and trivial.

Fair.